src/I18n/functions.php

```diff
@@ -226,3 +229,91 @@ function __dxn(
         ['_count' => $count, '_singular' => $singular, '_context' => $context] + $args
     );
 }
+
+/**
+ * Converts a value to a native DateTime object.
```

Comment:
```suggestion
 * Converts a value to a DateTime object.
```

The term "native" is generally used for PHP's classes, which isn't the case here and can be misleading.